sycl/source/handler.cpp

```diff
@@ -816,7 +816,11 @@ void handler::verifyUsedKernelBundle(const std::string &KernelName) {
     return;
 
   kernel_id KernelID = detail::get_kernel_id_impl(KernelName);
-  device Dev = detail::getDeviceFromHandler(*this);
+  device Dev;
```

Comment:
I think this default ctor isn't a no-op. Can we combine variable declaration and initialization in single statement? Either ternary operator or an "Immediately Invoked Function Expression".

Reply:
We updated it and now use a ternary operator.